omero/sysadmins/repository-move.txt

```diff
@@ -0,0 +1,85 @@
+.. _repository_move:
```

Comment:
A ref marker isn't really needed for an entire page of content, you can just use a `:doc:` link to refer to it from elsewhere.